FunctionSignatureMap.php

```diff
@@ -8570,27 +8569,27 @@
 'oci_lob_truncate' => ['bool', 'length'=>'int'],
 'oci_lob_write' => ['int', 'string'=>'string', 'length'=>'int'],
 'oci_lob_write_temporary' => ['bool', 'var'=>'string', 'lob_type'=>'int'],
-'oci_new_collection' => ['OCI_Collection', 'connection'=>'resource', 'tdo'=>'string', 'schema='=>'string'],
-'oci_new_connect' => ['resource', 'user'=>'string', 'pass'=>'string', 'db='=>'string', 'charset='=>'string', 'session_mode='=>'int'],
-'oci_new_cursor' => ['resource', 'connection'=>'resource'],
-'oci_new_descriptor' => ['OCI_Lob', 'connection'=>'resource', 'type='=>'int'],
-'oci_num_fields' => ['int', 'stmt'=>'resource'],
-'oci_num_rows' => ['int', 'stmt'=>'resource'],
-'oci_parse' => ['resource', 'connection'=>'resource', 'statement'=>'string'],
+'oci_new_collection' => ['OCI-Collection|false', 'connection'=>'resource', 'tdo'=>'string', 'schema='=>'string'],
```

Comment:
Phan is using the wrong name deliberately - it throws an FQSENException if it tries to analyze OCI-Collection or OCI-Lob(below), and doesn't have signatures for OCI-Lob